native/frontend/src/org/jetbrains/kotlin/resolve/konan/diagnostics/DefaultErrorMessagesNative.kt

```diff
@@ -25,6 +25,9 @@ private val DIAGNOSTIC_FACTORY_TO_RENDERER by lazy {
             "@SharedImmutable is applicable only to val with backing field or to property with delegation"
         )
         put(ErrorsNative.INAPPLICABLE_SHARED_IMMUTABLE_TOP_LEVEL, "@SharedImmutable is applicable only to top level declarations")
+        put(ErrorsNative.VARIABLE_IN_TOP_LEVEL_SINGLETON_WITHOUT_THREAD_LOCAL, "Variable in top level singleton without @ThreadLocal")
+        put(ErrorsNative.ENUM_THREAD_LOCAL_INAPPLICABLE, "@ThreadLocal isn't applicable to enum classes")
+        put(ErrorsNative.VARIABLE_IN_ENUM, "Variable in enum class can't be changed after initialization")
```

Comment:
Enum classes follow the same rules as `object` without `@ThreadLocal`, so I guess messages should probably be very similar in this part.